hi-ml-azure/src/health/azure/azure_util.py

```diff
@@ -793,3 +793,64 @@ def is_local_rank_zero() -> bool:
     global_rank = os.getenv(ENV_GLOBAL_RANK)
     local_rank = os.getenv(ENV_LOCAL_RANK)
     return global_rank is None and local_rank is None
+
+
+def download_from_datastore(datastore_name: str, prefix: str, output_path: Path,
+                            aml_workspace: Optional[Workspace] = None,
+                            workspace_config_path: Optional[Path] = None,
+                            overwrite: bool = False,
+                            show_progress: bool = False) -> None:
+    """
+    Download a file from an Azure ML Datastore that is registered within a given Workspace.
```

Comment:
Can you add more docu here? you write "download a file", but how is that specified? What is the role of the prefix? It would be great of have generally have more text here.